js/components/graph/Graph.js

```diff
@@ -1219,6 +1219,127 @@ export class Graph extends React.Component {
     return parents.every(isAllTrue)
   }
 
+  /**
+   * Renders a group of Bools
+   * @param {JSON} boolsJSON
+   * @param {object} boolsStatus
+   * @param {object} connections
+   * @return {JSX.Element}
+   */
+  renderBoolGroup = (boolsJSON, boolsStatus, connections) => {
+    return (
+      <g id="bools">
+        {Object.values(boolsJSON).map(boolJSON =>
+          generateBool(boolJSON, boolsStatus, connections)
```

Comment:
`generateBool` doesn't need to be a separate function, write the logic directly in here.

Reply:
And similarly for `generateEdge` below